emotes.go

```diff
@@ -207,28 +208,36 @@ func getUserIDs(names []string) []TwitchUser {
 }
 
 func getChannelEmotes(ID string) ([]EmoteInfo, map[string]map[string]string, error) {
-	resp, err := http.Get("https://api.twitchemotes.com/api/v4/channels/" + ID)
+	request, err := http.NewRequest("GET", fmt.Sprintf("https://api.twitch.tv/helix/chat/emotes?broadcaster_id=%s", ID), nil)
+	if err != nil {
+		log.Fatalln("Error generating new request:", err)
```

Comment:
Not sure if a Fatal fail is necessary/wanted here

Reply:
I agree, if it doesn't work, a wrapped error should be returned such as 
```go
fmt.Errorf("could not generate twitch channel emote request: %w", err)
```

Reply:
fair enough, I just followed the exact pattern from getUserIDs, but it would make more sense to do it this way in the emote section